app/webpack/stylesheets/components/_phase_banner.scss

```diff
@@ -0,0 +1,33 @@
+$laa-uat-colour: govuk-colour("orange");
+$laa-staging-colour: govuk-colour("turquoise");
+$laa-dev-colour: govuk-colour("pink");
+
+.uat {
+  .govuk-header__container {
+    border-bottom-color: $laa-uat-colour;
+  }
+
+  .govuk-tag.govuk-phase-banner__content__tag {
```

Comment:
If you keep the class on the header, this should be changed to:

`+ * .govuk-tag.govuk-phase-banner__content__tag {`

This way we don't need the class to be repeated on the phase tag.  

If moving the class to the body, this change is not needed.  

(Same as for both instances of the similar code below)

Reply:
👍 have moved class to body